src/runmode-dpdk.c

```diff
@@ -349,7 +349,7 @@ static void ConfigSetIface(DPDKIfaceConfig *iconf, const char *entry_str)
 
     retval = rte_eth_dev_get_port_by_name(entry_str, &iconf->port_id);
     if (retval < 0)
-        FatalError("Interface \"%s\": %s", entry_str, rte_strerror(-retval));
+        FatalError("Name of the interface (%s) is invalid: %s", entry_str, rte_strerror(-retval));
```

Comment:
this message suggests the name itself is invalid, but is that what is meant here? A name can be valid w/o being present in the system?

Reply:
No, it was meant to say that the interface does not exist. Will reword.